OsmAnd/src/net/osmand/plus/plugins/odb/OBDRemainingFuelWidget.kt

```diff
@@ -103,5 +115,10 @@ class OBDRemainingFuelWidget(
 				return app.getString(R.string.percent_unit)
 			}
 		}
+
+		fun next(): RemainingFuelMode {
```

Comment:
We have several enums with the same code, assuming that we slowly increase kotlin usage we could add universal methods for enums, something like this - package net.osmand.plus.utils

inline fun <reified T : Enum<T>> T.next(): T {
	val values = enumValues<T>()
	return values[(ordinal + 1) % values.size]
}

inline fun <reified T : Enum<T>> T.previous(): T {
	val values = enumValues<T>()
	return values[(ordinal - 1 + values.size) % values.size]
}